test/dummy/public/openapi.json

```
@@ -116,6 +116,20 @@
   },
   "components": {
     "schemas": {
+      "ProductRequestParameter":{
+        "type": "object",
+        "required": [
+          "product"
+        ],
+        "properties": {
+          "product": {
+            "type": "object",
+            "properties": {
+              "$ref": "#/components/schemas/Product"
+            }
+          }
+        }
```

`$ref` replaces itself to the referenced schema. Other documents need the same change.

```suggestion
        "properties": {
          "product": {
            "$ref": "#/components/schemas/Product"
          }
        }
```

---

Ya... I'm sure it was.

---

Fix 414a478